DataFormats/HGCDigi/src/classes_def.xml

```diff
@@ -1,5 +1,8 @@
 <lcgdict>
-   <class name="HGCSample"/>
+   <class name="HGCSample" ClassVersion="10">
+    <version ClassVersion="10" checksum="2120380856"/>
```

Comment:
new class versions should start from 3 https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning

Reply:
@slava77 please read https://github.com/cms-sw/cmssw/pull/28349#discussion_r345994831

Reply:
ok, thank you.